en/docs/install-and-setup/setup/setting-up-databases/overview.md

```diff
@@ -32,8 +34,9 @@ WSO2 products are shipped with scripts for creating the required tables in all t
 
 !!! Note
 
-    - To enable the default H2 database, add the below configurations to deployment.toml file in <APIM_HOME>/repository/conf and restart the server.
-          ```toml
-          [database_configuration]
-          enable_h2_console = true
-          ```
\ No newline at end of file
+    To enable the default H2 database, add the below configurations to deployment.toml file in <APIM_HOME>/repository/conf and restart the server.
```

Comment:
This has been incorrect from earlier. Shall we fix it too from this PR. It should be fixed as below.
"To eable browsing the H2 database from a web browser, add the below configurations to deployment.toml..."

Reply:
fixed.